schema.sql

```diff
@@ -0,0 +1,17 @@
+DROP DATABASE IF EXISTS sweethornbillbookstore;
+CREATE DATABASE sweethornbillbookstore;
+
+\c sweethornbillbookstore;
+
+
+CREATE TABLE books (
+  id SERIAL PRIMARY KEY,
+  title VARCHAR,
+  author VARCHAR,
+  genre VARCHAR,
+  description VARCHAR,
+  img_url VARCHAR
+);
+
+INSERT INTO books ( title, author, genre, description, img_url )
```

Comment:
i would separate out any hard coded entries into another file called 'fixture-data.sql' just so the only thing in schema.sql is what generates the database schema